hardemu/videoboard/sound.js

```diff
@@ -47,11 +66,18 @@ function Sound() {
         }
     }
 
+    this.reset = function() {
+        if(audio_ctx) {
```

Comment:
On reset, the `initialize` function will not be called so I suspect the sound will not work anymore after a reset in this case.
Executing `audio_gain.gain.value = 0;` instead should be enough to mute the voices on reset

Reply:
My bad, we do initialize on `io_write`